src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-size: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-size: 900;
+}
+
+html {
+  font-weight: 400;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.searchBar {
+  font-weight: 900;
+  font-family: inherit;
+  display: block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bigBar {
+  margin: 20px 0;
+  height: 70px;
+  background: url(./images/Search.svg) no-repeat;
+  background-position: 26px;
+  background-size: 19px;
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+.smallBar {
+  height: 42px;
+  background: url(./images/Search.svg) no-repeat;
+  background-position: 13px;
+  background-size: 11px;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.bigBar::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.smallBar::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-size: 14px;
```

Comment:
Do you need this rule ?

Reply:
Yes

Reply:
for small i need , for big i don't need it

Reply:
i need because i fail 2 npm tests

Reply:
need for fonts